tests/ctst/world/Zenko.ts

```diff
@@ -52,44 +83,41 @@ export default class Zenko extends World {
 
     private forceFailed = false;
 
+    readonly parameters: Record<string, unknown> = {};
+
     private cliMode: cliModeObject = CacheHelper.createCliModeObject();
 
     /**
      * @constructor
      * @param {Object} options - parameters provided as a CLI parameter when running the tests
      */
-    constructor(options: any) {
+    constructor(options: IWorldOptions) {
         super(options);
 
         // store service users credentials from world parameters
         if (this.parameters.ServiceUsersCredentials) {
-             Object.entries(JSON.parse(this.parameters.ServiceUsersCredentials))
-                .forEach((entry : [string, any]) => {
-                    Zenko.serviceUsersCredentials[entry[0] as string] = {
-                        AccessKeyId: entry[1].accessKey as string,
-                        SecretAccessKey: entry[1].secretKey as string,
-
+            Object.entries(JSON.parse(
+                this.parameters.ServiceUsersCredentials as string) as UserCredentials)
+                .forEach(entry => {
+                    Zenko.serviceUsersCredentials[entry[0]] = {
+                        AccessKeyId: (entry[1] as { accessKey: string }).accessKey,
+                        SecretAccessKey: (entry[1] as { secretKey: string }).secretKey,
                     };
                 });
         }
 
         // Workaround to be able to access global parameters in BeforeAll/AfterAll hooks
         CacheHelper.parameters = this.parameters;
-        this.cliMode.parameters = this.parameters;
+        this.cliMode.parameters = this.parameters as ClientOptions;
 
         if (this.parameters.AccountSessionToken) {
-            CacheHelper.ARWWI[CacheHelper.AccountName] = {
+            ((CacheHelper.ARWWI as Record<string, unknown>)[CacheHelper.AccountName]) = {
```

Comment:
The type of this object should be already known and should not be of type "unknown", but `Record<string, ClientOptions['AssumedSession']>` (according to your recent changes)